### PR TITLE
Tweak: Upgrade minimum required PHP version from 7.0 to 7.3 [ED-11983]

### PR DESCRIPTION
**Pull request by rami-elementor**
> By the end of the 3rd quarter, Elementor will raise the minimum required version to PHP 7.3, along with upgrading the recommended version to PHP 7.4.

Source: [Elementor’s Roadmap to Upgrade PHP Versions
](https://developers.elementor.com/elementors-roadmap-to-upgrade-php-versions/)

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -147,12 +147,12 @@ There is also a managed security [Bug Bounty program](https://go.elementor.com/w
 = Minimum Requirements =
 
 * WordPress 6.0 or greater
-* PHP version 7.0 or greater
+* PHP version 7.3 or greater
 * MySQL version 5.0 or greater
 
-= We recommend your host supports: =
+= Recommend Requirements =
```
Comment (yotamselementor) [inline readme.txt:153]:
```suggestion
= Recommended Requirements =
```

**Review by yotamselementor — DISMISSED** [review]

File: .github/workflows/phpunit.yml [hunk 2]
```diff
@@ -36,7 +36,7 @@ jobs:
       fail-fast: false
       matrix:
         wordpress_versions: ['nightly', 'latest', '6.0', '6.1', '6.2']
-        php_versions: ['7.0', '7.1', '7.2', '7.3', '7.4']
+        php_versions: ['7.3', '7.4', '8.0', '8.1', '8.2']
```
Comment (matipojo) [inline .github/workflows/phpunit.yml:39]:
```suggestion
        php_versions: ['7.3', '7.4']
```

**Review by matipojo — DISMISSED** [review]

**Comment by rami-elementor** [comment]
Hi @yotamselementor , 

@matipojo suggested to spilt all the test-related code changes to a separate task.

**Comment by yotamselementor** [comment]
> 

@rami-elementor I don't understand - what other changes do we have here that you want to split out?

**Review by arielk — APPROVED** [review]

**Review by TzviRabinovitch — APPROVED** [review]

**Comment by StyleShit** [comment]
what about composer.json?

**Comment by rami-elementor** [comment]
@StyleShit see https://github.com/elementor/elementor/pull/23619#issuecomment-1704295003 above

**Pull request merged**